app/src/main/res/values/strings.xml

```diff
@@ -39,4 +40,9 @@
     <string name="copy_to_clipboard">Copy to clipboard</string>
     <string name="qr_code">QR code</string>
     <string name="save_as_psbt_file">Save as .psbt file</string>
+    <string name="add_signer">Add Signer</string>
+
+    <string name="add_word_format">add word #%d</string>
```

Comment:
I will be better in some cases if we use `%$1d` instead.

Reply:
okay, but do you mean `%1$d` 😄